src/cmd-osbuild

```diff
@@ -8,10 +8,10 @@ dn=$(dirname "$0")
 
 print_help() {
     cat 1>&2 <<EOF
-Usage: coreos-assembler buildextend-${platform} --help
-       coreos-assembler buildextend-${platform} [--build ID]
+Usage: coreos-assembler osbuild --help
+       coreos-assembler osbuild [--build ID] --platforms qemu,metal,...
```

Comment:
Hmm, WDYT about:

```
cosa osbuild qemu metal ...
```

?

Seems more ergonomic and matches `cosa build`.

Reply:
OK. Will update to do it that way.